util/testUtils.js

```diff
@@ -22,3 +25,30 @@ export const middlewareDispatcher = middleware => (storeData, action) => {
 	return dispatched;
 };
 
+export const parseCookieHeader = (cookieHeader) => {
```

Comment:
I don't really have a strong preference. Sometimes with the arrow notation, it's slightly less clear at first glance, so I could also have a preference for explicit function blocks, but I don't truly care that much. 

I just have an OCD around readability when styles are intermixed.